docs/antrea-node-network-policy.md

```diff
@@ -105,6 +110,41 @@ spec:
           port: 22
 ```
 
+## Logs
+
+The `enableLogging` and `logLabel` options provide limited support for Node NetworkPolicies. Since Node NetworkPolicies
+are implemented using iptables, enabling `enableLogging` causes the Linux kernel to log information about all matching
+packets via the kernel log. However, Antrea cannot process these logs directly. Instead, these logs can be accessed
+through syslog, allowing you to filter and direct them to specific files using syslog syntax.
+
+For example, consider the Node NetworkPolicy `restrict-http-to-node` above:
+
+```text
+Sep  2 10:31:07 k8s-node-control-plane kernel: [6657320.789675] Antrea:In:Allow:allow-http:IN=ens224 OUT= MAC=00:50:56:a7:fb:18:00:50:56:a7:23:47:08:00 SRC=10.10.0.10 DST=192.168.240.200 LEN=60 TOS=0x00 PREC=0x00 TTL=64 ID=52813 DF PROTO=TCP SPT=57658 DPT=80 WINDOW=64240 RES=0x00 SYN URGP=0
+Sep  2 10:31:11 k8s-node-control-plane kernel: [6657324.899219] Antrea:In:Drop:default-drop:IN=ens224 OUT= MAC=00:50:56:a7:fb:18:00:50:56:a7:23:47:08:00 SRC=192.168.240.201 DST=192.168.240.200 LEN=60 TOS=0x00 PREC=0x00 TTL=64 ID=27486 DF PROTO=TCP SPT=33152 DPT=80 WINDOW=64240 RES=0x00 SYN URGP=0
+```
+
+In these logs, `Antrea:In:Allow:allow-http:` and `Antrea:In:Drop:default-drop:` are prefixes added by iptables using the
+`--log-prefix` parameter. The iptables log prefix is limited to 29 characters, as described in the
+[iptables-extensions manual](https://ipset.netfilter.org/iptables-extensions.man.html).
+
+The prefix format always includes `Antrea:[In|Out]:[Allow|Drop|Reject]:`, followed by the user-defined `logLabel` (if it
+is not empty). Due to the length restriction, any part that exceeds the 28-character limit (the last letter should be a
+space) will be truncated, as seen with `default-drop-others` being shortened to `default-drop`.
```

Comment:
you say the last character should be a space here, but the code is appending a colon, not a space?

Reply:
My bad. It should be a colon.